lib/omnibus/licensing.rb

```diff
@@ -95,6 +116,8 @@ def prepare
       FileUtils.rm_rf(output_dir)
       FileUtils.mkdir_p(output_dir)
       FileUtils.touch(output_dir_gitkeep_file)
+      FileUtils.rm_rf(cache_dir)
+      FileUtils.mkdir_p(cache_dir)
```

Comment:
Also, I think this change will require everyone to rebuild their git caches because snapshots made before this code is merged will not have the cache dir. I'm thinking we should coordinate merging this along with https://github.com/chef/omnibus/pull/704 and do that at a time that will be least disruptive for automate.